test/IDE/complete_function_builder.swift

```diff
@@ -93,3 +94,21 @@ struct EnumToVoidBuilder {
   static func buildBlock(_ :MyEnum, _: MyEnum, _: MyEnum) {}
 }
 func acceptBuilder(@EnumToVoidBuilder body: () -> Void) {}
+
+@_functionBuilder
+struct AnyBuilder {
+  static func buildBlock(_ components: Any...) -> Any { 5 }
+
+  static func #^IN_FUNCTION_BUILDER_DECL^#
```

Comment:
It might be good to add another test where we don't have `static func` to test the path where we insert it if it's missing.

Reply:
Good call, and I had a bug there! Fixed.